src/languages/alma-woocommerce-gateway.pot

```diff
@@ -437,14 +453,22 @@ msgid "Product price (%s): product badge injection failed."
 msgstr ""
 
 #: includes/class-alma-wc-settings.php:67
-msgid "Monthly and Deferred Payments with Alma"
+msgid "Pay in installments with Alma"
+msgstr ""
+
+#: includes/class-alma-wc-settings.php:68 includes/class-alma-wc-settings.php:70 includes/class-alma-wc-settings.php:72
+msgid "Fast and secure payment by credit card"
+msgstr ""
+
+#: includes/class-alma-wc-settings.php:69
+msgid "Buy now, Pay later with Alma"
 msgstr ""
 
-#: includes/class-alma-wc-settings.php:68
-msgid "Pay in deferred or multiple monthly payments with your credit card."
+#: includes/class-alma-wc-settings.php:71
+msgid "Spread your payments with Alma"
```

Comment:
If you change a key, you have to update all related `.po` files as we said during our meeting on friday

I show 2 removed keys in this commit (you can find revisions history [in crowdin](https://almapay.crowdin.com/u/projects/17/files).

Please update translations keys into `.po` files